truffe2/templates/base.html

```diff
@@ -455,9 +455,22 @@
                                     {% endifhasright %}
                                     {% ifhasright 'accounting_tools.models.Invoice' user 'LIST' %}
                                         <li id="menu-compta-invoice">
-                                            <a href="{% url 'accounting_tools.views.invoice_list' %}"><i class="fa fa-lg fa-fw fa-pencil-square-o"></i> {% trans "Factures" %}</a>
+                                            <a href="{% url 'accounting_tools.views.invoice_list' %}"><i class="fa fa-lg fa-fw fa-pencil-square-o"></i> {% trans "Factures (Client)" %}</a>
                                         </li>
                                     {% endifhasright %}
+                                    
+  			            {% ifhasright 'accounting_tools.models.ProviderInvoice' user 'LIST' %}
```

Comment:
Les tabs sont pétées par ici

Reply:
@the-glu , fixed

Reply:
Sorry for that shitty problems :/ and thank again